src/broker/broker.c

```diff
@@ -849,7 +849,7 @@ receiver_thr_f (void *arg)
 
       if (strncmp (cas_req_header, "ST", 2) == 0)
 	{
-	  int status = -1;
+	  int status = -2;
```

Comment:
The meaning of initialized value (-2) is not clear. It should be defined as a enum value (in cas_function.h) such as `FN_STATUS_NONE = -2`.